style.css

```diff
@@ -0,0 +1,166 @@
+body {
+  margin: 20px;
+  margin-left: 35px;
+  text-align: center;
+  color: black;
+  scroll-behavior: smooth;
+}
+a:link, a:visited {
+  text-decoration: none;
+  color: black;
+  font-size: 25px;
+  font-family: permanent marker;
+}
+
+.alphabet{
+  font-size: 10px;
+  line-height: 120%;
+}
```

Comment:
To achieve a fixed or sticky element, you set the position to be fixed, and then specify top/bottom/right/left values. Here's an example with a sticky header:
```
.title-area {
  position: fixed;
  top: 0;
  width: 100%;
  background: white;
}

.introduction {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}
```

For the fixed element to not overlap your other content, you can add some margin to the top of the next element which in this case is the class alphabet: `margin-top: 270px;`